src/tests/nunit.runner.tests.wp81/MainPage.xaml.cs

```diff
@@ -53,7 +53,11 @@ public MainPage()
                 //TcpWriterParameters = new TcpWriterInfo("192.168.0.108", 13000),
 
                 // Creates a NUnit Xml result file on the host file system using PCLStorage library.
-                CreateXmlResultFile = false
+                CreateXmlResultFile = false,
```

Comment:
We've removed WP81 since you started this PR. Someone will need to resolve the merge conflicts. Sorry.